ui/app/components/sidebar/nav/cluster.js

```diff
@@ -25,4 +26,19 @@ export default class SidebarNavClusterComponent extends Component {
     // should only return true if we're in the true root namespace
     return this.namespace.inRootNamespace && !this.hasChrootNamespace;
   }
+
+  get showSecretsSync() {
+    // always show for HVD managed clusters
+    if (this.flags.isHvdManaged) return true;
```

Comment:
I like calling this out here, but it does get checked on the `this.version.hasSecretSync`

Reply:
I added it here because per our discussion you mentioned that this work should show for ALL HVD managed clusters regardless of activation status.